core/admin/EE_Admin_List_Table.core.php

```diff
@@ -806,8 +810,8 @@ public function extra_tablenav($which)
         } else {
             echo '<div class="list-table-bottom-buttons alignleft actions">';
             foreach ($this->_bottom_buttons as $type => $action) {
-                $route         = $action['route'] ?? '';
-                $extra_request = $action['extra_request'] ?? '';
+                $route         = isset($action['route']) ? $action['route'] : '';
+                $extra_request = isset($action['extra_request']) ? $action['extra_request'] : '';
```

Comment:
They are equivalent and null coalescing operator is cleaner and more readable. Any reason to switch to classic syntax?

Reply:
i agree but master branch still allows PHP 5.6